garrysmod/lua/vgui/dcolorbutton.lua

```diff
@@ -27,9 +27,9 @@ function PANEL:IsDown()
 
 end
 
-function PANEL:SetColor( color, displayTooltip )
+function PANEL:SetColor( color, hideTooltip )
 
-	if ( displayTooltip == true or displayTooltip == nil ) then
+	if ( hideTooltip == false or hideTooltip == nil ) then

```

Comment:
```lua
if ( hideTooltip ) then
```

Reply:
Wouldn't that change it so the tooltip is only visible when it is true? Shouldn't it he true for no Tooltip?

Reply:
I think he meant ``if ( !hideTooltip ) then``

Reply:
Yes that makes sense